resources/assets/tconstruct/lang/es_es.lang

```diff
@@ -0,0 +1,992 @@
+item.tconstruct.moms_spaghetti.tooltip3=§oJusto como mamá solía hacerlos.
+stat.spaghetti.uses.name=Ñoms
+stat.spaghetti.nourishment.name=Llenitud
+stat.spaghetti.saturation.name=Saturación
+modifier.spaghetti_sauce.name=con Salsa
+modifier.spaghetti_sauce.desc=§o¡Cómo mamá los hacía!§r\nIncrementan la saturación.
+modifier.spaghetti_meat.name=con Carne
+modifier.spaghetti_meat.desc=§o¡Toda la Carne!§r\nIncrementa la llenitud.
+
+item.tconstruct.spaghetti.hard.name=Palos de Trigo Imposiblemente Duros
+item.tconstruct.spaghetti.hard.tooltip=Totalmente incomestibles.\nHay que suabizarlos. Ni el sudor de tus manos es suficiente.\nQuizás deberían remojarse en agua un poco.
+item.tconstruct.spaghetti.soggy.name=Fideos Aguados
+item.tconstruct.spaghetti.soggy.tooltip=Ahora estan mojados. Incluso tus brazos de espagueti pueden doblarlos.\nHay que ponerlos a secar.
+item.tconstruct.spaghetti.cold.name=Fideos Secos Fríos
+item.tconstruct.spaghetti.cold.tooltip=Mejor, pero aun saben asqueroso\nCalentarlos debería hacerlos comestibles.\nQuizás sean suficiente para endurecer un poco esas rodillas débiles.
+
+#################
+#   Entities    #
+#################
+
+entity.tconstruct.blueslime.name=Slime Azúl
+
+#################
+#    Liquids    #
+#################
+
+fluid.tconstruct.stone.name=Piedra Chamuscada
+fluid.tconstruct.obsidian.name=Obsidiana Fundida
+fluid.tconstruct.clay.name=Arcilla Fundida
+fluid.tconstruct.dirt.name=Tierra Líquida
+fluid.tconstruct.iron.name=Hierro Fundido
+fluid.tconstruct.gold.name=Oro Fundido
+fluid.tconstruct.pigiron.name=Acerdo Fundido
+fluid.tconstruct.cobalt.name=Cobalto Fundido
+fluid.tconstruct.ardite.name=Ardita Fundida
+fluid.tconstruct.manyullyn.name=Manyulín Fundido
+fluid.tconstruct.knightslime.name=Acero de Slime Fundido
+fluid.tconstruct.emerald.name=Esmeralda Fundida
+fluid.tconstruct.glass.name=Cristal Fundido
+fluid.tconstruct.blood.name=Sangre
+fluid.tconstruct.milk.name=Leche
+fluid.tconstruct.blueslime.name=Slime Azúl Líquido
+fluid.tconstruct.purpleslime.name=Slime Púrpura Líquido
+fluid.tconstruct.brass.name=Latón Fundido
+fluid.tconstruct.copper.name=Cobre Fundido
+fluid.tconstruct.tin.name=Estaño Fundido
+fluid.tconstruct.bronze.name=Bronce Fundido
+fluid.tconstruct.zinc.name=Cinc Fundido
+fluid.tconstruct.lead.name=Plomo Fundido
+fluid.tconstruct.nickel.name=Níquel Fundido
+fluid.tconstruct.silver.name=Plata Fundida
+fluid.tconstruct.electrum.name=Electro Fundido
+fluid.tconstruct.steel.name=Acero Fundido
+fluid.tconstruct.aluminum.name=Aluminio Fundido
+fluid.tconstruct.alubrass.name=Latón de Aluminio Fundido
+
+#################
+#    Potions    #
+#################
+
+potion.carry=Cangureando
+
+#################
+#   Materials   #
+#################
+# material.<identifier>.name
+# material.<identifier>.prefix
+# material.wood.prefix=Wooden %s - Used for stuff like "Wooden Pickaxe" if no specific name is given
```

Comment:
See here for changing the order of the materials in tools

Reply:
Oh, thanks. That makes sense in why Wood Tools have the correct identification.